web_dynamic/README.md

```diff
@@ -0,0 +1,39 @@
+# 0x06. AirBnB clone - Web dynamic
+
+===
+
+## Description
+
+---
+Repository to study the following Topic: RESTful <-> Javascript <-> HTML
+
+![Web Dynamic Graphic](https://github.com/jdrestre/pictures-holberton-projects/blob/master/0x06AirBnBclone_web_dinamic/hbnb_step5.png)
+
+### General
+
+- How cool it is to request your own API
+- How to modify an HTML element style
+- How to get and update an HTML element content
+- How to modify the DOM
+- How to make a GET request with jQuery Ajax
+- How to make a POST request with jQuery Ajax
+- How to listen/bind to DOM events
+- How to listen/bind to user events
+
+## Task Project
+
+---
+File Name|Task Name|Task Description
+---|---|---
+[**`AirBnB_clone_v4/`**](https://github.com/jdrestre/AirBnB_clone_v4/)|**0. Last clone!**|Update the repository name to AirBnB_clone_v4
+[**`0-hbnb.py`**](https://github.com/jdrestre/AirBnB_clone_v4/blob/master/web_dynamic/0-hbnb.py), [**`templates/0-hbnb.html`**](https://github.com/jdrestre/AirBnB_clone_v4/blob/master/web_dynamic/templates/0-hbnb.html)|**1. Cash only**|Write a script that starts a Flask web application
+
+## Author
+
+---
+
+- Simón Parra - [Github](https://github.com/Simonster04) / [Twitter](https://twitter.com/Simonster04)
```

Comment:
Mr Juan, Authors section is not working well in the Readme when you visualize it on Github. One solution you can implement is to take a look the original file and apply the changes you need to in that file.

Reply:
Done! I have already updated so that each author is seen in a single line